anthos-multi-cloud/AWS/vars.sh

```diff
@@ -0,0 +1,21 @@
+export CLUSTER_NAME=aws-cluster-qa
```

Comment:
Do we need to have these vars.sh files? I believe if anything, provide steps on how to get the appropriate variables in the readme?

Reply:
Oh yes, those vars.sh files were added by accident!

Reply:
I just removed them in the latest commit